docs/guide/process_data.rst

```diff
@@ -321,19 +321,14 @@ parameters. The custom defined plots will automatically be added to the result p
     variable of the string array is shown on the x-axis. The second variable is shown on the y-axis. The third is
     represented as different lines, and the fourth as different marker styles::
 
-        stringArray comparePlot "x_variable" "y_variable" ["series 1 variable"] ["series 2 variable"]
+        stringArray comparePlot "x_variable" "y_variable" ["series 1 variable"] ["series 2 variable"] ["filter1"] ["filter2"] ...
 
     .. image:: ./resources/ComparisonPlot.png
         :width: 400
-        :alt: SP
```

Comment:
@martin-neugebauer : I've removed this line as I thought "SP" is not very informative to a user that doesn't see the image. I suspect this is something that was just copied over from somewhere else?

Reply:
Yes, this was definitely copied.